BookReader/BookReader.js

```diff
@@ -3889,14 +3864,13 @@ BookReader.prototype.swipeMousemoveHandler = function(event) {
 };
 
 BookReader.prototype.swipeMouseupHandler = function(event) {
```

Comment:
Same here with the `swipe`/`drag` naming.

Reply:
thanks for tipping me over that fence! :done: